### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by idom3923** [UnamSanctam/UnamWebPanel#208]
for example, if i put the endpoint in multiple exe's, will it show on panel?

**Comment by UnamSanctam** [comment]
Yes, if you compile many miners with your panel in the "API Endpoint URL" then they will all show up in the web panel when they run.
